pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1401,40 +1316,38 @@ func podToMemberPod(pod *v1.Pod, includeIP, includePodRef bool) *controlplane.Gr
 	return memberPod
 }
 
-func externalEntityToGroupMember(ee *v1alpha1.ExternalEntity) *controlplane.GroupMember {
+func externalEntityToGroupMember(ee *v1alpha2.ExternalEntity) *controlplane.GroupMember {
 	memberEntity := &controlplane.GroupMember{}
-	for _, endpoint := range ee.Spec.Endpoints {
-		var namedPorts []controlplane.NamedPort
-		for _, port := range endpoint.Ports {
-			namedPorts = append(namedPorts, controlplane.NamedPort{
-				Port:     port.Port,
-				Name:     port.Name,
-				Protocol: controlplane.Protocol(port.Protocol),
-			})
-		}
-		ep := controlplane.Endpoint{
-			IP:    ipStrToIPAddress(endpoint.IP),
-			Ports: namedPorts,
-		}
-		memberEntity.Endpoints = append(memberEntity.Endpoints, ep)
+	var namedPorts []controlplane.NamedPort
```

Comment:
maybe we can initialize here with the len(Spec.Ports)

Reply:
Done